apps/www/app/layout.tsx

```diff
@@ -11,6 +11,7 @@ const PostHogPageView = dynamic(() => import("@/providers/posthog/PostHogPageVie
   ssr: false,
 });
 export const metadata: Metadata = {
+  metadataBase: new URL("https://unkey.com/"),
```

Comment:
When overriding the default, we recommend using environment variables to compute the URL. This allows configuring a URL for local development, staging, and production environments.

Setting this to our URL doesn't truly fix the issue, we should be reading from process.env.X and then setting it that way. 

Once this is set the images should be set to a relative path versus absolute.

Reply:
can we use our zod env to parse this and set a default?

```ts
baseUrl: z.string().default("https://unkey.com")
```

?

Reply:
![image](https://github.com/user-attachments/assets/ca89206c-7837-4c69-9fd7-ea0b0b8b448e)
this is how we are talking about it right? Also I think I should configure it to unkey URL if there is nothing in the .env

Reply:
> can

Like this right?

![image](https://github.com/user-attachments/assets/071fdc58-75c5-4cdf-bdbb-f7d423fb0762)